receiver/redisreceiver/README.md

```diff
@@ -0,0 +1,89 @@
+# Redis Receiver
+
+### Overview
+
+The Redis receiver is designed to retrieve Redis INFO data from a single Redis
+instance, build metrics from that data, and send them to the next consumer at a
+configurable interval.
+
+Status: beta
+
+### Details
+
+The Redis INFO command returns information and statistics about a Redis
+server (see [https://redis.io/commands/info](https://redis.io/commands/info) for
+details).
+The Redis receiver extracts values from the result and converts them to open
+telemetry metrics. Details about the metrics produced by the Redis receiver
+can be found by browsing [metric_functions.go](metric_functions.go).
+
+For example, one of the fields returned by the Redis INFO command is
+`used_cpu_sys` which indicates the system CPU consumed by the Redis server,
+expressed in seconds, since the start of the Redis instance.
```

Comment:
Nit: please use consistent line spacing for paragraphs (the first 2 paragraphs do not have an empty line between).

Reply:
Done (it's just one paragraph).